src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -254,6 +254,16 @@ public static IServiceCollection AddMvc([NotNull] this IServiceCollection servic
             return WithControllersAsServices(services, controllerTypes.Select(type => type.AsType()));
         }
 
+        public static IServiceCollection AddMvcLocalization([NotNull] this IServiceCollection services)
```

Comment:
This is temporary. I will move this method to Localization package once I create one.